text-generation-inference/server/text_generation_server/version.py

```diff
@@ -1,4 +1,4 @@
 from pkg_resources import parse_version
 
-__version__ = "0.1.0.dev0"
+__version__ = "0.1.0.dev2"
```

Comment:
ok for now, later on we will need to change this since devX with X>0 is invalid for setuptools.